programs/ttp_oracle/src/processor.rs

```diff
@@ -43,20 +46,97 @@ impl Processor {
 
     Ok(())
   }
+
+  pub fn process_handle_response(accounts: &[AccountInfo], response: &Response) -> ProgramResult {
+    // clear the account data
+    let accounts_iter = &mut accounts.iter();
+    let oracle_account = next_account_info(accounts_iter)?;
+    let mut data = oracle_account.try_borrow_mut_data()?;
+    data.copy_from_slice(&[0u8; Request::LEN]);
+    // send a cross program invocation to the second account
+    let client_program_account = next_account_info(accounts_iter)?;
+    let data = &mut [0u8; (Response::LEN + 1 as usize)];
```

Comment:
lets add that `+ 1 as usize` thats for the response data type flag either to the Response struct or some other struct that wraps Response